controllers/auth_controller.go

```diff
@@ -10,30 +10,32 @@ import (
 )
 
 type RegisterInput struct {
-	Username    string `json:"username" binding:"required"`
-	Password    string `json:"password" binding:"required"`
-	UserType    string `json:"usertype" binding:"required"`
-	Firstname   string `json:"firstname" binding:"required"`
-	Lastname    string `json:"lastname" binding:"required"`
-	Birthdate   string `json:"birthdate" binding:"required"`
-	CitizenId   string `json:"citizenId" binding:"required"`
-	Gender      string `json:"gender" binding:"required"`
-	PhoneNumber string `json:"phoneNumber" binding:"required"`
-	Address     string `json:"address" binding:"required"`
-	AvatarUrl   string `json:"avatarUrl"`
+	Username    string  `json:"username" binding:"required"`
+	Password    string  `json:"password" binding:"required"`
+	UserType    string  `json:"usertype" binding:"required"`
+	Firstname   string  `json:"firstname" binding:"required"`
+	Lastname    string  `json:"lastname" binding:"required"`
+	Birthdate   string  `json:"birthdate" binding:"required"`
+	CitizenId   string  `json:"citizenId" binding:"required"`
+	Gender      string  `json:"gender" binding:"required"`
+	PhoneNumber string  `json:"phoneNumber" binding:"required"`
+	Address     string  `json:"address" binding:"required"`
+	AvatarUrl   string  `json:"avatarUrl"`
+	lat         float32 `json:"lat" binding:"required"`
+	lng         float32 `json:"lng" binding:"required"`
```

Comment:
These two should be changed to be a public variable using PascalCase

```suggestion
	Lat         float32 `json:"lat" binding:"required"`
	Lng         float32 `json:"lng" binding:"required"`
```

Reply:
which will require more change in the code as well please check carefully